include/s3select_oper.h

```diff
@@ -885,7 +880,7 @@ class value
   value& operator/(value& v)
   {
     if (v.is_null() || this->is_null()) {
-      v.set_nan();
+      v.setnull();
```

Comment:
ok,
any arithmetic operation with NULL should result with UNKNOWN (value.type = NULL)